tests/validate_contributors_json.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import json
+from pathlib import Path
+import jsonschema
+
+def validate_contributor_metadata(contributors_filepath:Path):
+    """Check if the the specified file is error-free JSON, and its content matches the 
+    all-contributors schema. If not, then raise a JSONDecodeError or ValidationError respectively.
+
+    Keyword Arguments:
+        contributors_filepath {Path} -- Filepath of the JSON file containing contributors' metadata
+    """
+
+    # Load the schema specification that the all-contributors JSON must follow
+    contributor_schema = None
+    with open("./tests/all-contributors.schema.json") as f:
```

Comment:
```suggestion
    with open("./tests/all-contributors.schema.json") as f:
```

Given that we've already imported pathlib, I'd rather use that library to construct a path dynamically rather than rely on hardcoding it here